rviz_rendering/src/rviz_rendering/point_cloud.cpp

```diff
@@ -551,17 +500,16 @@ void PointCloud::addPoints(Point* points, uint32_t num_points)
       current_vertex_count = 0;
 
       vertex_size = op->vertexData->vertexDeclaration->getVertexSize(0);
-      fptr = (float*)((uint8_t*)vdata);
+      fptr = reinterpret_cast<float *>(reinterpret_cast<uint8_t *>(vdata));
```

Comment:
Not 100% sure about this one, I think either the interior reinterpret cast is not needed or it should be a static cast?

Reply:
The interior cast is actually completely unnecessary. This will be addressed in the follow-up PR we are preparing where we do the refactoring. For now, this was just meant to shut up the linter. But if you like, we can of course edit it here already.